lib/internal/modules/esm/assert.js

```diff
@@ -81,7 +81,7 @@ function validateAssertions(url, format,
         // `type` wasn't specified at all.
         throw new ERR_IMPORT_ASSERTION_TYPE_MISSING(url, validType);
       }
-      handleInvalidType(url, importAssertions.type);
+      return handleInvalidType(url, importAssertions.type);
```

Comment:
I think this name is now misleading: I expect something named "handle" to handle it—meaning there's no follow-up _required_.

Reply:
Good point. I'll think of a better name (later, when I get back to this). Suggestions welcome.

Reply:
I went with `createHandleInvalidTypeError()` instead.

Reply:
Whoops, "handle" is a verb and not a noun. I'm new here.

Reply:
I don't think the original name was misleading after all. "handleInvalidType()` always throws. The "return" is added just to make it clear to JSDoc that the code can't get beyond that point. 

```
function createHandleInvalidTypeError(url, type) {
  // `type` might have not been a string.
  validateString(type, 'type');

  // `type` might not have been one of the types we understand.
  if (!ArrayPrototypeIncludes(supportedAssertionTypes, type)) {
    throw new ERR_IMPORT_ASSERTION_TYPE_UNSUPPORTED(type);
  }

  // `type` was the wrong value for this format.
  throw new ERR_IMPORT_ASSERTION_TYPE_FAILED(url, type);
}
```